### PR TITLE
[stable] 1.45 release

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,166 @@
...
+  you can now write the following;
+  ```rust
+  for ch in 'a'..='z' {
+      print!("{}", ch);
+  }
+  println!();
+  // Prints "abcdefghijklmnopqrstuvwxyz"
+  ```
+- [`OsString` now implements `FromStr`.][71662]
+- [The `saturating_neg` method as been added to all signed integer primitive
```
Comment (MikailBag) [inline RELEASES.md:50]:
```suggestion
- [The `saturating_neg` method has been added to all signed integer primitive
```

Comment (Mark-Simulacrum) [reply RELEASES.md:50]: Feel free to file a PR against master -- it won't make it into this stable release, but it should make it into the next one. Thanks!

**Pull request merged**